backend/main.py

```
@@ -18,14 +23,35 @@
 async def hello():
     return {"message": "Hello, World!", "status": "success"}
 
 @app.get("/")
-async def root():
-    return {"message": "api works!"}
+def read_root(db: Session = Depends(get_db)):
+    # Your database queries here
+    return {"message": "Connected to Neon!"}
+
+@api.get("/db-info")
+def get_db_info(db: Session = Depends(get_db)):
```

Ruff (0.13.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)